### DISCUSSION TITLE
Screen navigation

### DISCUSSION THREAD

**Discussion by thebino** [AndroidDev-social/DodoForMastodon#47] [General]
To navigate through the app there are some patterns and libraries around. Lets collect some of them here with Pros/Cons and decide which to use.

- [Navigating with Compose](https://developer.android.com/jetpack/compose/navigation) (only works on Android)
- [Decompose](https://github.com/arkivanov/Decompose) (multiplatform support)

**Comment by bitPogo** [comment]
I heard good things about [Voyager](https://github.com/adrielcafe/voyager) - it supports Android/JVM.

**Comment by thebino** [comment]
⛔️ This discussion is closed. Please vote or [check for results](https://github.com/AndroidDev-social/MastodonCompose/discussions/56)
